src/love.cpp

```diff
@@ -26,6 +26,10 @@
 
 #ifdef LOVE_BUILD_EXE
 
+#if SDL_VERSION_ATLEAST(3, 0, 0)
+#include "SDL_main.h"
```

Comment:
Should this use system-relative includes (`<SDL_main.h>` instead of `"SDL_main.h"`)?

Reply:
sure